data/transactions/logic/assembler.go

```diff
@@ -2873,10 +2877,18 @@ func checkByteImmArgs(cx *EvalContext) error {
 	return err
 }
 
-func parseSwitch(program []byte, pos int) (targets []int, nextpc int, err error) {
```

Comment:
pos uint?  maybe i missed something.

Reply:
It is very annoying, in Go, to use `uint` extensively, when dealing with array/slice indexes.  You can't, for example, compare a uint to `len(x)`, so your code ends up littered with casts.